src/Storages/SelectQueryInfo.h

```diff
@@ -184,7 +184,7 @@ struct SelectQueryInfo
     SeekToInfoPtr seek_to_info_of_right_stream; /// Rewind info for right streaming store in streaming query
 
     /// if is true, means need sort backfill input by event time (ascending)
-    bool require_in_order_backfill = true;
+    bool require_in_order_backfill = false;
```

Comment:
Do we need disable watermark during backfill when in order backfill is false ? Do we emit tumble windows during backfill or global aggregation ?

Reply:
For 1) , if control by `emit_aggregated_during_backfill ` , when it is false, we don't generate watermark  for historical data
(But there are still some limitations need to be resolved in future, when exists subquery with join/aggr, we can not get the historical flag from subquery input)

For 2) by default `emit_aggregated_during_backfill  = false` , don't emit the aggr result

Reply:
you can see this logic here.
https://github.com/timeplus-io/proton/blob/037555c1bf7f1568ee1e3c3e974a809d5e407ade/src/Interpreters/InterpreterSelectQuery.cpp#L3495-L3509C2